compiler/rustc_feature/src/removed.rs

```diff
@@ -140,6 +140,9 @@ declare_features! (
     (removed, const_fn, "1.54.0", Some(57563), None,
      Some("split into finer-grained feature gates")),
 
+    /// Allows `#[doc(include = "some-file")]`.
+    (removed, external_doc, "1.54.0", Some(44732), None, Some("use #[doc = include_str!(\"filename\")] instead, which handles macro invocations")),
```

Comment:
Nit: it still makes sense to keep the 100 character line limit even if rustfmt doesn't enforce it in macros.